playbooks/install_monitoring_stack.yml

```diff
@@ -32,7 +32,7 @@
         mode: 0640
       loop:
         - "{{ playbook_dir }}/../files/dashboards/subnets.json.j2"
-      when: grafana_subnets_dashboard_variables is defined
+      when: grafana_subnets_dashboard_variables is defined and (grafana_subnets_dashboard_variables|length>0)
```

Comment:
I think this will fail if `grafana_subnets_dashboard_variables` is undefined. Using `default` might work and we can probably simplify the whole condition this way.

Reply:
Ok I'm on it

Reply:
But if the variable `grafana_subnets_dashboard_variables` is undefined it should just skip this step because the first part of the when condition `grafana_subnets_dashboard_variables is defined` is not respected right ?
I've just run the playbook as it is without touching the variable and it works just fine....

Reply:
It's all good now